Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -687,9 +649,9 @@ AppDelegateViewModelOutputs {
     deepLinkFromNotification
       .observeValues { _ in AppEnvironment.current.koala.trackNotificationOpened() }
 
-    self.applicationIconBadgeNumber = self.applicationLaunchOptionsProperty.signal
+    self.applicationIconBadgeNumber = self.applicationWillEnterForegroundProperty.signal
```

Comment:
Can we merge `applicationLaunchOptionsProperty` with `applicationWillEnterForegroundProperty` so the badge is cleared both from fresh launches and background launches?